api/envoy/extensions/compression/common/decompressor/v3/decompressor.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package envoy.extensions.compression.common.decompressor.v3;
+
+import "google/protobuf/any.proto";
+
+import "udpa/annotations/migrate.proto";
+import "udpa/annotations/status.proto";
+import "udpa/annotations/versioning.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.compression.common.decompressor.v3";
+option java_outer_classname = "DecompressorProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Decompressor]
+
+message DecompressorLibrary {
+  // Decompressor library specific configuration. See the supported libraries for further
```

Comment:
Where can the "supported libraries" documentation be found? (Might make sense to provide a hint here)